book/website/communication/peer-review/peer-review-guidance.md

```diff
@@ -24,10 +24,11 @@ Are you planning to submit your own work there?
 Is this journal an [Open Access](https://the-turing-way.netlify.app/reproducible-research/open/open-access.html) journal?
 Beware of [predatory journals](https://predatoryjournals.com/journals/).
 
-1. Check if you are the right person to review the paper. 
+1. Check if this paper is of interest to review and if you are the right person to review the paper. 
 Reviewers do not need to be experts in all respects. 
```

Comment:
The editor may ask you to focus on a specific aspect of the paper. Alternatively, you may qualify your review with an indication of where you focused your review, if you do not feel you have the necessary expertise to evaluate all aspects of the paper.

Reply:
Added, thanks! Are you already listed as a contributor for content?